util/src/main/java/io/kubernetes/client/util/PagerParams.java

```diff
@@ -0,0 +1,30 @@
+package io.kubernetes.client.util;
+
+public class PagerParams {
+
+  private Integer limit;
+  private String _continue;
+
+  public PagerParams() {}
+
+  public PagerParams(Integer limit, String _continue) {
+    this.limit = limit;
+    this._continue = _continue;
+  }
+
+  public Integer getLimit() {
+    return limit;
+  }
+
+  public void setLimit(Integer limit) {
```

Comment:
how about making the `limit` an immutable value here? (which is, settable by constructor but didn't have setter/getter). otherwise, what's the use-case of changing page size dynamically?